### PR TITLE
Add ability to get mysql password from file

### PR DESCRIPTION
**Pull request by skimpax**
Hello,

To avoid putting cleartext password in docker-compose file, use the facility to put the password in a file, as used by docker secrets.
The file path is to be set set in MYSQL_PASS_FILE.
In case of use in docker secret, path shall be /run/secrets/my_docker_secret_name

Regards.

### CODE REVIEW

File: backup.sh [hunk 1]
```diff
@@ -8,12 +12,16 @@ echo "=> Backup started at $(date "+%Y-%m-%d %H:%M:%S")"
...
+      && [[ "$db" != "performance_schema" ]] \
+      && [[ "$db" != "mysql" ]] \
+      && [[ "$db" != "sys" ]] \
+      && [[ "$db" != _* ]]
   then
     echo "==> Dumping database: $db"
     FILENAME=/backup/$DATE.$db.sql
     LATEST=/backup/latest.$db.sql.gz
-    if mysqldump --single-transaction -h "$MYSQL_HOST" -P "$MYSQL_PORT" -u "$MYSQL_USER" -p"$MYSQL_PASS" "$db" $MYSQLDUMP_OPTS > "$FILENAME"
+    if mysqldump --single-transaction -h "$MYSQL_HOST" -P "$MYSQL_PORT" -u "$MYSQL_USER" -p"$MYSQL_PASS" "$db" "$MYSQLDUMP_OPTS" > "$FILENAME"
```
Comment (fradelg) [inline backup.sh:24]: I think MYSQLDUMP_OPTS may include double quotes and this will break some cases. Can you leave it unquoted as it was before?

**Comment by fradelg** [comment]
Hi! Thank you for your contribution. If you can address my comment I will be happy to merge your PR with master

**Comment by skimpax** [comment]
Sorry for the delay. Shall be OK now.

**Comment by fradelg** [comment]
No worries, thanks for your contribution

**Pull request merged**